wordpress-mobile/WordPress-Android
# [Blogging prompts v1] Change number of answers text color in dashboard and introduction dialog

#16801 

To test:
Dashboard card:
- Run JP variant
- Select a site that has blogging prompts enabled
- Open the home screen
- Check the prompt card: the number of answers label should have a regular text color and should not look like a button

Introduction dialog:
- Run JP variant
- Select a site that has blogging prompts enabled
- Open the home screen
- Tap on "MENU" tab
- Tap on "Site Settings"
- Tap on "Reminders and prompts"
- Tap on "Set reminders" in case there were no reminders set for this site yet
- Select any day of the week to display the "Include daily prompt" section
- Tap on the "?" button
- Check the prompt card: the number of answers label should have a regular text color and should not look like a button

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
